Source/MonoGame.Extended/Collections/Poolable.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MonoGame.Extended.Collections
+{
+    public abstract class Poolable : IPoolable
```

Comment:
The problem with using abstract classes vs interfaces is that a class can only derive from a single base class, but it can implement many interfaces. So in this case, I don't see why there's any benefit to having the abstract class (unless we plan to add to it later?)